tests/integration/issues/github_1229/test_sharding_empty_index.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import numpy as np
+
+from jina import Flow, Document
+
+
+# do not delete
+
+
+def test_sharding_empty_index(random_workspace_name):
+    print("WORKSPACE =", random_workspace_name)
+    os.environ['JINA_TEST_1229_WORKSPACE'] = os.path.abspath(random_workspace_name)
+    num_shards = 2
+
+    f = Flow() \
+        .add(
+        uses='vectorindexer.yml',
+        shards=1,
```

Comment:
u need to index with 2 shards. I guess it is still on progress right?

Reply:
Yes it was to shove the code to me for debugging.

Reply:
Yep, done now.